docs/src/main/sphinx/connector/query-passthrough-warning.fragment

```diff
@@ -0,0 +1,10 @@
+The native query passed to the underlying data source is required to return a
+table as a result set. This can be a one column table with one row. No
+validation or security checks of these queries are performed by Trino. The
+data source performs these tasks using its own configuration.
+
+You can use passthrough queries to read data; however, queries that write data
```

Comment:
This implies that writes are not possible unless the underlying database supports `RETURNING`. This seems like too strong a statement to make. It seems more likely that in most databases, it would be possible to execute the write on the underlying database, even if the Trino query ends up failing due to no result set.

Reply:
So instead of saying we don't support writes, we need to better describe that it may appear to fail but still be processed since that's a worse outcome than just failing outright.

Suggest: "You can use passthrough queries to read data; however, queries that write data typically cannot be used since they do not return a table. If supported by the underlying data source, you can enable such queries by..."
->
"You can use passthrough queries to read data; however, queries that write data typically do not return a table, so when a write operation is passed Trino declares that the query failed even if the database performs the operation. If supported by the underlying data source, you can avoid this outcome by..."